content/ar/courses/350/blog-project/setup.md

```diff
@@ -1,56 +1,71 @@
 ---
-# Page title
-title: Django Project and App Setup
 
-# Title for the menu link if you wish to use a shorter link title, otherwise remove this option.
-linktitle: Setup
+# عنوان الصفحة
 
-# Date page published
-date: 2021-03-23
+العنوان: مشروع جانغو واعداد التطبيق
 
-# Academic page type (do not modify).
-type: book
+# عنوان رابط القائمة اذا كنت ترغب في استخدام عنوان ارتباط اقصر, والا فقم بازالة هذا الخيار.
 
-# Position of this page in the menu. Remove this option to sort alphabetically.
-weight: 1
+عنوان الرابط: اعداد
 
-draft: False
+# تاريخ نشر الصفحة
+
+التاريخ: 32-3-2021
+
+# عنوان الصفحة الاكاديمية (لاتعدل عليه)
+
+النوع: كتاب
+
+# مكان هذة الصفحة في القائمة. قم بازالة هذا الخيار للترتيب ابجديا.
+
+الوزن: 1
+
+المسودة: خطأ
```

Comment:
@2172116630 please do not translate the list above the --- mark
 only title and linktitle ... they should be like this:

```
title: مشروع جانغو واعداد التطبيق
linktitle: اعداد
```

everythin above --- is data used by the system and should not be translated. Please fix it and push the fix to the same branch